src/compiler/generate.ml

```diff
@@ -57,7 +57,7 @@ let generate ctx tctx ext actx =
 	end;
 	if actx.interp then
 		Std.finally (Timer.timer ["interp"]) MacroContext.interpret tctx
-	else if com.platform = Cross then
+	else if platform_define com.platform = "cross" then
```

Comment:
This change doesn't look good, why can't we stick to proper variant matching here?

Reply:
Ah yeah, wanted to make this comment too earlier

Reply:
I try to avoid checking against `""` these days but it's fine here.